site/content/docs/5.0/backwards-compatibility.md

```diff
@@ -54,6 +54,11 @@ These custom Arizona Bootstrap classes are now deprecated:
 The `.nav-tabs-lg` custom Arizona Bootstrap class is now deprecated.
 
 
+### UL Triangles
+
+The `.ul-triangles` custom Arizona Bootstrap class has beed replaced by `.az-list-triangles`.
```

Comment:
It might be good to explicitly mention the deprecation here. Also, very nitpicky, but this class isn't actually from the Components section. I was wondering about this for the note about Tables (since that is under "Content") and I ended up putting it in its own section below. But I think this is fine--just maybe something to consider for #1586.

```suggestion
The `.ul-triangles` custom Arizona Bootstrap class has been deprecated and replaced by `.az-list-triangles`.
```